src/templates/article-latest.js

```diff
@@ -0,0 +1,565 @@
+import React from 'react'
+import { getImage } from 'gatsby-plugin-image'
```

Comment:
I don't think we need this anymore as QueryImage handles this underneath. 🙏

Reply:
Amended in latest commit